lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -228,6 +253,12 @@ private void migrateIdentity(
                     e,
                     batchId,
                     pageSummary);
+            auditService.sendAuditEvent(
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_EVCS_MIGRATION_FAILURE,
+                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                            new AuditEventUser(reportItem.getHashUserId(), "", "", ""),
+                            new AuditExtensionsEvcsFailedMigration(batchId, 0, e.getMessage())));
```

Comment:
Instead of using the error's message, I'd be tempted to use something more static - like "Failed to fetch VCs from tactical store". We're logging the error already, so will be able to get those details if needed. As we're catching all exceptions, it's possible that the message recorded like this won't actually be that useful.